docker-compose.yml

```
@@ -0,0 +1,61 @@
+version: "3.4"
+
+services:
+  web:
+    build:
+      context: .
+      args:
+        - RAILS_MASTER_KEY=${RAILS_MASTER_KEY}
+        - RAILS_ENV=${RAILS_ENV}
+    environment:
+      TEST_DATABASE_URL: postgresql://nokul:nokul@db/nokul_test
+      DEVELOPMENT_DATABASE_URL: postgresql://nokul:nokul@db/nokul_development
+      REDIS_URL: redis://redis:6379
+    ports:
+      - "80:3000"
+    depends_on:
+      - db
+      - redis
+  redis:
+    image: "redis:alpine"
+  db:
+    image: "postgres:11.1"
+    environment:
+      POSTGRES_USER: nokul
+      POSTGRES_PASSWORD: nokul
+  sidekiq:
+    build:
+      context: .
+      args:
+        - RAILS_MASTER_KEY=${RAILS_MASTER_KEY}
+        - RAILS_ENV=${RAILS_ENV}
+    environment:
+      TEST_DATABASE_URL: postgresql://nokul:nokul@db/nokul_test
+      DEVELOPMENT_DATABASE_URL: postgresql://nokul:nokul@db/nokul_development
+      REDIS_URL: redis://redis:6379
+    command: bash -c "bundle exec sidekiq -q high -q low"
```

```suggestion
    command: sh -c "bundle exec sidekiq -q high -q low"
```

(mikro optimizasyon)